### DISCUSSION TITLE
NavigationBar's Secondary Command in dark mode on Android

### DISCUSSION THREAD

**Discussion by WenHong-Lau** [unoplatform/uno#18233] [Q&A] [answered]
I'm not using Material Design. After removing it, I believe the default is Fluent Design?

There are no issues with WinAppSDK, the background switches to Dark Mode as expected. However, on Android, while the text changes to Dark Mode (white), the background remains unchanged, staying in Light Mode.
![image](https://github.com/user-attachments/assets/6b5c1c3a-f933-42d1-ab34-89933d56ac7c)

**Comment by morning4coffe-dev** [comment]
Hey @WenHong-Lau,

Thank you for your question. Could you provide a small repro for me to test this out? I wasn't able to replicate with my project. For using the Fluent/WinUI controls, the MergedDictionaries should look something like: https://github.com/morning4coffe-dev/recurrents/blob/0070cd0984002932eb9608abc7f349a112041bed/src/Recurrents/App.xaml#L15

**Comment by WenHong-Lau** [comment]
Hi @morning4coffe-dev,

This is out of the box from the template where I picked the Fluent Design, I've only added the AppBarButton in Secondary Commands
[UnoApp5.zip](https://github.com/user-attachments/files/17041634/UnoApp5.zip)
![Screenshot 2024-09-18 171024](https://github.com/user-attachments/assets/f5faeb67-0eeb-4367-9d0c-23f3971a264e)
![Screenshot 2024-09-18 171130](https://github.com/user-attachments/assets/95a8dc39-dd08-4995-b2db-e2ac64a820f6)

**Reply by morning4coffe-dev** [reply]
Thank you for taking the time to repro, @WenHong-Lau. I can confirm, I am seeing the same behavior on newer Android versions. I will open an issue to track this. If you try using the native styles, does this get fixed? https://platform.uno/docs/articles/external/uno.toolkit.ui/doc/controls/NavigationBar.html#modes

**Reply by WenHong-Lau** [reply]
No, seems like it still the same after applying the style

**Comment by morning4coffe-dev** [comment, marked as answer]
Here is the issue to track this: https://github.com/unoplatform/uno.toolkit.ui/issues/1250
